CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 CHANGELOG
 ==========================
 
+## 1.3.2 (`1030270`)
+
+- **Fix:** Cron entries that use "Day of the week" run schedule will now correctly generate readme entries
```

Comment:
Please add reference to PR. Check previous changes for example.

Reply:
Think it's just a simple # like issue references, wonder how GH would resolve it if there was an issue and PR with the same number 🤔